microsoft-edge/web-platform/site-impacting-changes.md

```diff
@@ -15,10 +15,10 @@ The web is constantly evolving to improve the user experience, security, and pri
-| Disallow synchronous XmlHttpRequest in page dismissal | Shipping in M82 | This change is happening in the Chromium project, on which Microsoft Edge is based.  For more information, including the planned timeline by Google for this change, please review the [Chrome Platform Status entry][ChromePlatformStatus4664843055398912].  Based on enterprise and developer feedback, the Microsoft Edge team is shipping this feature in Microsoft Edge later than Chrome.  Matching Chrome, Edge offers a Group Policy to disable this change until Edge 88.  |  
-| Disable TLS/1.0 and TLS/1.1 by default | Shipping in M82 | The `edge://flags/#show-legacy-tls-warnings` flag may be set to cause Edge to show a non-blocking "Not Secure" notice when loading pages that require legacy TLS protocols.  The [SSLMinVersion][DeployedEdgePoliciesSSLMinVersion] Group Policy permits re-enabling of TLS/1.0 and TLS/1.1; the policy remains available until Edge 88.  |  
+| Cookies default to `SameSite=Lax` | Canary/Dev experimentation starting in v82 | This change is happening in the Chromium project, on which Microsoft Edge is based. For more information, including the planned timeline by Google for this change, please review the [Chrome Platform Status entry][ChromePlatformStatus5088147346030592]. Based on user and developer feedback, the Microsoft Edge team is shipping this feature in Microsoft Edge later than Google.  |  
+| Referrer Policy: Default to `strict-origin-when-cross-origin` | Canary/Dev experimentation starting in v79 | This change is happening in the Chromium project, on which Microsoft Edge is based. For more information, including the planned timeline by Google for this change, please review the [Chrome Platform Status entry][ChromePlatformStatus6251880185331712]. Based on user and developer feedback, the Microsoft Edge team is shipping this feature in Microsoft Edge at the same time or later than Google.  |  
+| Disallow synchronous XmlHttpRequest in page dismissal | Shipping in v83 | This change is happening in the Chromium project, on which Microsoft Edge is based. For more information, including the planned timeline by Google for this change, please review the [Chrome Platform Status entry][ChromePlatformStatus4664843055398912]. Based on enterprise and developer feedback, the Microsoft Edge team is shipping this feature in Microsoft Edge later than Chrome. Matching Chrome, Edge offers a Group Policy to disable this change until Edge 88.  |  
+| Disable TLS/1.0 and TLS/1.1 by default | Shipping in v83 | To help discover impacted sites, you may set the `edge://flags/#show-legacy-tls-warnings` flag to cause Edge to show a non-blocking "Not Secure" notice when loading pages that require legacy TLS protocols. The [SSLMinVersion][DeployedEdgePoliciesSSLMinVersion] Group Policy permits re-enabling of TLS/1.0 and TLS/1.1; the policy remains available until Edge 88.  |  

```

Comment:
can we change:
...TLS/1.1; the policy remains available until Edge 88.  |  
to:
...TLS/1.1. This policy will be available in Microsoft Edge up to v88.  |

Reply:
Moved to a suggestion note.